lib/3scale/backend/transactor.rb

```diff
@@ -99,7 +99,8 @@ def validate(oauth, provider_key, report_usage, params, request_info)
           # to derive which limits affect directly or indirectly the
           # metrics for which authorization is requested.
           hierarchy:       extensions[:hierarchy] == '1',
-          flat_usage:      extensions[:flat_usage] == '1'
+          flat_usage:      extensions[:flat_usage] == '1',
```

Comment:
freeze?

Reply:
yes, we should be freezing the literals :)

Reply:
:cold_face: